war/src/test/java/io/jenkins/pluginhealth/scoring/http/ScoreAPITest.java

```diff
@@ -247,12 +260,141 @@ void shouldGet304WhenNoNewScoring() throws Exception {
+
+        // @formatter:off
+        mockMvc.perform(get("/api/scores"))
+                .andExpectAll(
+                        status().isOk(),
+                        content().contentType(MediaType.APPLICATION_JSON),
+                        header().string("ETag", equalTo("\"" + scoreP1Date.toEpochSecond() + "\"")),
+                        content()
+                                .json(
+                                        """
+                    {
+                        'plugins': {
+                            'plugin-1': {
+                                'value': 100,
+                                'date': "%s",
+                                'previousScore': 100,
+                                'details': {
+                                    'scoring-1': {
+                                        'value': 100,
+                                        'weight': 1,
+                                        'components': [{
+                                            'value': 100,
+                                            'weight': 1,
+                                            'reasons': ['There is no active security advisory for the plugin.']
+                                        }]
+                                    }
+                                }
+                            },
+                            'plugin-2': {
+                                'value': 100,
+                                'date': "%s",
+                                'previousScore': 10,
+                                'details': {
+                                    'scoring-1': {
+                                        'value': 100,
+                                        'weight': 1,
+                                        'components': [{
+                                            'value': 100,
+                                            'weight': 1,
+                                            'reasons': ['There is no active security advisory for the plugin.']
+                                        }]
+                                    }
+
+                                    }
+
+                            }
+
+                        },
+                        'statistics': {
+                            'average': 87.5,
+                            'minimum': 50,
+                            'maximum': 100,
+                            'firstQuartile': 100,
+                            'median': 100,
+                            'thirdQuartile': 100
+                        }
+                    }
+                    """
+                                                .formatted(
+                                                        scoreP1Date.format(DateTimeFormatter.ISO_OFFSET_DATE_TIME),
+                                                        scoreP1Date.format(DateTimeFormatter.ISO_OFFSET_DATE_TIME)),
+                                        false));
+    }
```

Comment:
reactivate the formatter before closing the method.

Reply:
@alecharp Cannot format manually once I edit the file, spotless causing build failure 
<img width="1596" alt="Screenshot 2024-04-03 at 10 05 02 PM" src="https://github.com/jenkins-infra/plugin-health-scoring/assets/60972989/1eab115b-7ec7-4310-91d5-7ad28c00d20f">

Reply:
you need to add `// @formatter:on` before the end of the method.

Reply:
Done @alecharp